### ISSUE TITLE
Symfony Flex (3.3) - Dependency with Twig

### ISSUE THREAD

**Issue by matrix818181** [FriendsOfSymfony/FOSRestBundle#1772]
When I want to enable exception handlers, I get the following exception :

The service "fos_rest.exception.twig_controller" has a dependency on a non-existent service "templating.engine.twig".

**Comment by ramoshka** [comment]
Try add line to your config
```
fos_rest:
    exception:
        enabled: true
        exception_controller: 'fos_rest.exception.controller:showAction'
```
If you do not define exception controller and have TwigBundle enabled `fos_rest.exception.twig_controller:showAction` will be used`
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/80a41f571250ad4c14f7e36c267eb968575884d8/DependencyInjection/FOSRestExtension.php#L331

**Comment by MichaelHindley** [comment]
@ramoshka I'm seeing this error with the following config:

```yml
   exception:
      enabled: true
      exception_controller: 'AppBundle\Port\Rest\RequestValidation\RequestValidationExceptionListener::render'
```

It's on 3.4 though, should I open a new issue?

**Comment by xabbuh** [comment]
I guess both issues have the same cause. So no need for a new issue IMO.

**Comment by rsaenen** [comment]
Having the same issue with Symfony 4 and Flex.

**Comment by Sharom** [comment]
Check your bundles registration order. Try to load `FOSRestBundle` after the `TwigBundle`:

```php
 <?php
// bundles.php
return [
    // ...
    Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
    FOS\RestBundle\FOSRestBundle::class => ['all' => true],
];
```

It helped for me.

**Comment by ahundiak** [comment]
Given that the templating engine is no longer installed and enabled by default, why not modify the TwigExceptionPass to just check for the twig service instead of templating.engine.twig?  Seem like it should still work.  You lose the old style twig template path support but that should not be a problem.

**Comment by LLCDBDBBSysUser** [comment]
I have the same problem. Fix is available in #1945. Would be nice if it would get merged.
